int32 frontDoorPos
int32 samplerPos
bool samplerStatus
bool weighingPanStatus
